backend/dataall/cdkproxy/stacks/policies/quicksight.py

```diff
@@ -7,6 +7,8 @@ class QuickSight(ServicePolicy):
     def get_statements(self):
         return [
             iam.PolicyStatement(
```

Comment:
Quicksight resources cannot be tagged, that's why I added the teamname in the resource string